modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,326 @@
+** Max connection life time: Maximum connection lifetime for the connection pool
+** Max connection life time time unit: Time unit for the Maximum connection life time field
+** Max connection idle time: Maximum connection idle time for the connection pool
+** Max connection idle time unit: Time unit for the Maximum connection idle time field
+
+== Changed Operations, Parameters, and Return Types
+
+The following table shows changes to operation names, input parameters, and return types:
+
+[%header%autowidth.spread]
+|===
+|MongoDB 5.x Operation | Changes in MongoDB 6.0
+
+| Count documents a| Input parameters are now:
+
+ * `Collection name` (String)
+ * `Query` (JSON)
+
+Return type is now a Long with the count result.
+
+| Create Collection a| Input parameters are now:
+
+ * `Collection name` (String)
+ * `Max objects` (Integer)
+ * `Collection size` (Integer)
+ * `Collection size data unit` (TimeUnit)
+
+| Exists collection | Operation is now called Collection exists.
+
+| Create file from payload a|
+
+* Operation is now called Create file.
+* Return type is now a JSON.
+
+
+| Create index a| Input parameters are now:
+
+ * `Collection name` (String)
+ * `Field name` (String)
+ * `Sort order` (Enum: ASC or DESC)
+
+| Dump a| Input parameters are now:
+
+* `Output directory` (String)
+* `Output name prefix` (String)
+* `Oplog` (no change)
+* `Operation timeout` (Integer)
+* `Operation timeout unit` (TimeUnit)
+
+Return type is now a List<String> that points to the created files. Each string is a filePath.
+
+| Execute command a|
+
+* Input parameter is `Command` (JSON).
+* Return type is now a JSON that contains the result of the command.
+
+| Find documents a| Input parameters are now:
+
+ * `Collection name` (String)
+ * `Query` (JSON)
+ * `Fields` (String)
+ * `Sort by` (JSON)
+ * `Page size` (Integer)
+ * `Limit` (Integer)
+
+This operation supports pagination. Each item returned as a JSON.
+
+| Find files a|
+
+* Input parameters are now:
+ ** `Query` (JSON)
+ ** `Sort` (JSON)
+* Return type is now a JSON list.
+
+| Get file content a|
+
+* Input parameter is now `File id` (JSON).
+* Return type is now a stream with the binary file content. This operation also returns the file information as attributes.
+
+| Insert document | Return type is now a entire JSON object with the `_id` object of the created document.
+
+| Insert documents | Return type is now a Bulk Operation Result, which contains a JSON that lists each created record and its status.
+
+| List collections | Return type is now a List<String> that contains the names of the collections.
+
+| List indices | Operation is now called List indexes.
+
+| Map reduce objects a|
+
+* Operation is now called Map reduce.
+* Return type is now a JSON that represents the output specified in the reduce function.
+
+| Remove documents a|
+
+* Input parameters are now:
+ ** `Collection name` (String)
+ ** `Query` (JSON)
+* Return type is now a Long with the count result.
```

Comment:
GLOBAL in this text: if a proper noun/name is preceded by an article, it must be followed by a common noun object.
I will stop marking this. Please take a look throughout.